index.js

```diff
@@ -419,6 +419,21 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a unary function and transforms it into a unary function in another
+  //. setting.
```

Comment:
How about this?

_Takes a unary function and returns the corresponding unary function in another setting._

I'd love to say more, but this is as much of a description as my current understanding permits.